### DISCUSSION TITLE
Function to create tables from common datatypes

### DISCUSSION THREAD

**Discussion by Adilius** [Textualize/rich#1006] [Ideas] [answered]
Is there plans to create functions that automatically creates tables from different datatypes such as pandas.dataframe or directly from files such as .arff or .csv?

I recently used rich tables to present data in a dataframe and thought perhaps it could be a built in constructor

**Comment by willmcgugan** [comment, marked as answer]
Not currently. I wouldn’t want to introduce a dependency that most people wouldn’t need.

ideally this could be done in a third party lib, or integrated in to other libraries. Rich provides the tools, but it probably shouldn’t try to presume how people will want to render their data.
